etc/docker/elyra/start-elyra.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+export NB_PORT=${NB_PORT:-8888}
+export GATEWAY_HOST=${GATEWAY_HOST:-localhost}
+export KG_URL=${KG_URL:-http://${GATEWAY_HOST}:${NB_PORT}}
+export KG_HTTP_USER=${KG_HTTP_USER:-jovyan}
+export KG_REQUEST_TIMEOUT=${KG_REQUEST_TIMEOUT:-120}
+export KERNEL_USERNAME=${KG_HTTP_USER}
+
+echo "Gateway URL: " ${JUPYTER_GATEWAY_URL}
+echo "JupyterLab port: " ${NB_PORT}
+echo "Kernel user: " ${KERNEL_USERNAME}
+
+echo "${@: -1}"
+
+
+exec /usr/local/bin/start-singleuser.sh $*
```

Comment:
Based on this, there is no option to **not** use Enterprise Gateway.  Seems like we should allow for not using EG.

I would suggest using `JUPYTER_GATEWAY_URL` and make the user bring that as an env.

```
docker run ... -e JUPYTER_GATEWAY_URL=http://foo.bar:8888 ...
```

Things like request timeout (and user) can be setup unconditionally...
```
export JUPYTER_GATEWAY_REQUEST_TIMEOUT=${JUPYTER_GATEWAY_REQUEST_TIMEOUT:-120}
export JUPYTER_GATEWAY_HTTP_USER=${JUPYTER_GATEWAY_HTTP_USER:-${NB_USER}}
```

Reply:
+1 on the not requiring EG option

Reply:
yes, and I was able to use that in hub both ways

Reply:
i didn't play much with it as a standalone image tough.

Reply:
ok - sounds good.  I think once the new variables are updated, you should be good.